packages/mui-docs/src/MarkdownElement/MarkdownElement.tsx

```diff
@@ -174,8 +174,6 @@ const Root = styled('div')(
         boxShadow: 'none',
         fontWeight: 'inherit',
         position: 'relative',
-        display: 'flex',
-        alignItems: 'center',
```

Comment:
Do you have any why these attributes have been added in? 🤔

Reply:
I do not manage to find when exactly we added it, because of the migration to the `@mui/docs` which pertubate the git blame

From what I remember, we struggled to have nice vertical alignment between the "anchor link", "comment link", and the title constent. But seems it's working well. So I suspect a DOM modification we did and forgot to remove this display flex

Reply:
Thanks for clarifying. 👌 
Hopefully, this doesn't break some other sections that we are not aware of. 🙈